include/openssl/tls1.h

```diff
@@ -351,6 +351,9 @@ SSL_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_DEBUG_CB,(void (*)(void))cb)
 # define SSL_set_tlsext_debug_arg(ssl, arg) \
 SSL_ctrl(ssl,SSL_CTRL_SET_TLSEXT_DEBUG_ARG,0, (void *)arg)
 
+# define SSL_get_tlsext_status_type(ssl, arg) \
```

Comment:
okay, but it's hard to tell that getters return int values but stuff complex things into ```*arg``` for pointers.  might be nice to always try to stuff a return value, and simply return a status.